webhooks/README.md

```diff
@@ -53,14 +53,84 @@ function isSignatureValid(fidelHeaders, payload, secret, url) {
 ```
 
 To prevent replay attacks where a valid payload and its signature is intercepted and re-transmitted, you can use the `x-fidel-timestamp` header and confirm that the timestamp is not too old. We recommend you validate the requests in a 5 minute gap. In case of retries, a new signature and timestamp are generated for each new request.
+---
+
+## Custom headers
+Fidel allows you to define custom HTTP headers that are passed in the webhook POST requests sent to you application.
+Custom headers can be defined when creating a new webhook in the dashboard or by using the [Webhooks API](https://reference.fidel.uk/v1/reference#create-webhook-brand) by setting the optional `headers` object with a key-value pair. You can use the [Update  Webhook](https://reference.fidel.uk/reference#create-webhook-program) endpoint and submit an empty `headers` object in order to remove the custom headers from the webhook.
```

Comment:
`You can use the [Update  Webhook](https://reference.fidel.uk/reference#create-webhook-program) endpoint and submit an empty `headers` object in order to remove the custom headers from the webhook.`

1. Should we not link to all endpoints that allow headers? There are three.
2. In this block we use as example `https://reference.fidel.uk/v1/reference#create-webhook-brand` and on L62 we use `https://reference.fidel.uk/reference#create-webhook-program`.
3. I would reword this without specifying the PUT, otherwise it might seem it's the only place that you can send empty headers. Something like: `You can reset (or remove) your custom headers from the webhook by submitting an empty headers object.`

Reply:
to 3., would you use any other method than `PUT` to update a webhook?

Reply:
You can send POST with empty headers and it will be accepted.

Reply:
I updated the link of the first to the create hook by program and the second to the PUT.